client/components/container/container-toolbar-directive.js

```diff
@@ -40,6 +40,18 @@ goog.scope(function() {
 
         /** @export {!DashboardService} */
         this.dashboardSvc = dashboardService;
+
+        /**
+         * Prompts the user for confirmation, then deletes the selected container.
+         * @export
+         */
+        this.removeSelectedContainer = function() {
+          if (!window.confirm('Are you sure you want to delete this container?')) {
```

Comment:
There isn't a user-useful ID for a container or widget, and other that the URL, it's not exposed.  I'll update the text to "The selected container/widget will be deleted."  The selected item is also visibly highlighted, so it should be clear.